src/lib/webpack-config.js

```diff
@@ -208,6 +216,37 @@ export default env => {
 			}
 		}),
 
+		// TypeScript
+		customConfig({
+			module: {
+				loaders: [
+					{
+						enforce: 'pre',
+						test: /\.tsx?$/,
+						use: [
+							{
+								loader: resolve(__dirname, './npm-install-loader'),
+								options: {
+									modules: ['typescript', 'awesome-typescript-loader'],
+									save: true
+								}
+							},
+							{
+								loader: 'awesome-typescript-loader',
+								options: {
+									sourceMap: true,
+									useBabel: true,
+									babelOptions: createBabelConfig(env),
```

Comment:
I wonder - if we enabled `babel-loader` for `/.tsx?/` files, could we drop the `useBabel` and config here?  TS files would get passed through `awesome-typescript-loader` (thanks to `enforce:pre`) and then later transformed from ES2017 to JS-of-yesteryear via the existing `babel-loader` rule.

Thoughts?